Library/ViewModels/PledgePaymentMethodsViewModel.swift

```diff
@@ -56,12 +63,20 @@ public final class PledgePaymentMethodsViewModel: PledgePaymentMethodsViewModelT
       .values()
       .map { $0.me.storedCards.nodes }
 
-    self.reloadPaymentMethods = Signal.merge(
+    let card = Signal.merge(
       storedCards,
       self.newCreditCardProperty.signal.skipNil().map { card in [card] }
     )
     .scan([]) { current, new in new + current }
 
+    let cardValues = Signal.combineLatest(
+      card,
+      availableCardTypes,
+      project.map { $0.location.displayableName }
+    ).map { $0 as CardViewValues }
+
+    self.reloadPaymentMethods = cardValues
```

Comment:
Here is where we could move even more logic out of the view controller – something we could do is instead of returning `CardViewValues`, return an array of `PledgeCreditCardValue`:

```
    self.reloadPaymentMethods = Signal.combineLatest(
      cards,
      availableCardTypes,
      project.map { $0.location.displayableName }
    ).map { cards, availableCardTypes, projectLocation in
      return cards.map { card in
        let isEnabled = availableCardTypes.contains(card.type.rawValue)

        return (card, isEnabled, projectLocation)
      }
    }
```

Then we can test that given a list of stored cards, and a list of `availableCardTypes` in a project, that the output is exactly what we're feeding into each `PledgeCreditCardViewModel` to configure the card view.